client/odie/message/index.tsx

```diff
@@ -73,15 +107,57 @@ const ChatMessage = ( { message, isLast, messageEndRef }: ChatMessageProps ) =>
 				</div>
 			) }
 			{ message.type === 'introduction' && (
-				<>
-					<div className="odie-introduction-message-content">
-						<img src={ WapuuAvatar } alt="Wapuu profile" className="odie-chatbox-message-avatar" />
-						<div className="odie-chatbox-introduction-message">{ message.content }</div>
-					</div>
-				</>
+				<div className="odie-introduction-message-content">
+					<img src={ WapuuAvatar } alt="Wapuu profile" className="odie-chatbox-message-avatar" />
+					<div className="odie-chatbox-introduction-message">{ message.content }</div>
+				</div>
 			) }
 		</div>
 	);
+
+	useEffect( () => {
+		const backdropCopy = backdropRef.current;
+		if ( isFullscreen ) {
+			backdropRef.current?.addEventListener( 'click', handleBackdropClick );
+		}
+		return () => {
+			backdropCopy?.removeEventListener( 'click', handleBackdropClick );
+		};
+	}, [ isFullscreen ] );
+
+	const fullscreenContent = (
+		<div className="odie-fullscreen" ref={ backdropRef } onClick={ handleBackdropClick }>
+			<div onClick={ handleContentClick }>{ messageContent }</div>
+		</div>
+	);
+
+	const isBubbleType = [ 'message', 'introduction', 'placeholder', 'help-link' ].includes(
+		message.type
+	);
+
+	const messageContentWithTail = (
+		<>
+			{ messageContent }
+			{ isBubbleType && ! isUser && (
+				<img
+					src={ WapuuChatTail }
+					className="odie-chatbox-message-wapuu-tail"
+					alt="Customer chat tail"
+				/>
+			) }
+			{ isBubbleType && isUser && (
+				<img
+					src={ CustomerChatTail }
+					className="odie-chatbox-message-customer-tail"
+					alt="Wapuu chat tail"
```

Comment:
We probably want to translate these alts.

Reply:
We are going to drop this things, won't be used at all at the end I think, they are quite probably be deleted any time soon